index.d.ts

```diff
@@ -61,6 +63,32 @@ declare const tempy: {
 	*/
 	directory(): string;
 
+	/**
+	Write a string/buffer/stream to a random temp file.
+
+	@example
+	```
+	import tempy = require('tempy');
+
+	await tempy.write('🦄');
+	//=> '/private/var/folders/3x/jf5977fn79jbglr7rk0tq4d00000gn/T/2f3d094aec2cb1b93bb0f4cffce5ebd6'
+	```
+	*/
+	write(fileContent: string | Buffer | NodeJS.ReadableStream, options?: tempy.Options): Promise<string>
```

Comment:
It needs to handle the same as `fs.writeFile`, which includes `<string> | <Buffer> | <TypedArray> | <DataView>`, + our stream handling.

Reply:
Fixed!